package.json

```diff
@@ -8,19 +8,19 @@
     "fuzzysort": "^1.1.4",
     "gh-pages": "^3.1.0",
     "js-sha256": "^0.9.0",
-    "mdbreact": "mdbreact-4.26.1.tgz",
     "mdb-react-sortable": "mdb-react-sortable-4.26.0.tgz",
+    "mdbreact": "mdbreact-4.26.1.tgz",
     "node-sass": "^4.12.0",
     "obelisk.js": "^1.2.2",
-    "react-avatar-editor": "^11.0.7",
+    "react-avatar-editor": "^11.0.9",
```

Comment:
Why have you upgraded to 11.0.9? I had some issues with this version.
Ref: https://github.com/mosch/react-avatar-editor/issues/340

Reply:
For me the version 11.0.7 did not work, that's why I upgraded it.

Reply:
Strange..

Maybe we should fork it and fix it for them.